shared/common-adapters/icon.desktop.js

```diff
@@ -72,7 +72,7 @@ class Icon extends Component<Props, void> {
         }
       : null
 
-    const hasContainer = (this.props.onClick && this.props.style) || isFontIcon
+    const hasContainer = (onClick && this.props.style) || isFontIcon
```

Comment:
this and below can likely be the same as in master
this should also respect the noClickableBox prop